tools/loader/ImageClassifier.cpp

```diff
@@ -198,9 +208,8 @@ buildAndCompileAndGetInAndOutPair(Loader &loader, PlaceholderBindings &bindings,
   // Get the Tensor from the Placeholder that the final expected Softmax writes
   // into at the end of image inference.
   Placeholder *SMPH = EXIT_ON_ERR(LD->getSingleOutput());
-  Tensor *SMT = bindings.get(SMPH);
```

Comment:
If we're not returning the Tensor do we still need to allocate it (above)?

Reply:
We still need the backing tensor, we just access it via the PH now.